metadata-ingestion/src/datahub/ingestion/source/fivetran/__init__.py

```diff
@@ -0,0 +1 @@
+from datahub.ingestion.source.fivetran.fivetran import FivetranSource
```

Comment:
this probably isn't necessary - make init empty, and make setup.py point directly at the fivetran source class instead of re-exporting it here